packages/babel-plugin-transform-es2015-shorthand-properties/README.md

```diff
@@ -30,7 +30,7 @@ var cat = {
 
 ```js
 var cat = {
-  getName: function () {
+  getName: function getName() {
```

Comment:
I know but it seems weird to say that the output for this specific transform is what the preset outputs and not the plugin?